src/components/buttons/_private/ButtonBase/ButtonBase.scss

```diff
@@ -206,24 +273,52 @@ $disabled-text-dark: $gray-dark50;
 	@include setThemeVar('--Button__TextDecoration', underline);
 }
 
+.ButtonBase__TextDecoration_Underline_Hover {
+	@include setThemeVar('--Button__TextDecoration_Hover', underline);
+}
+
 .ButtonBase__LetterSpacing_Normal {
 	letter-spacing: normal;
 }
 
 .ButtonBase__LetterSpacing_Tracked {
+	@include tracking(5);
+}
+
+.ButtonBase__LetterSpacing_Extra {
 	@include tracking(40);
 }
 
+.ButtonBase__Display_Inline {
+	display: inline-flex;
+}
+
 .ButtonBase__FontSize_XSmall {
-	font-size: 11px;
+	font-size: 1.1rem;
 }
 
 .ButtonBase__FontSize_Small {
-	font-size: 12px;
+	font-size: 1.2rem;
 }
 
 .ButtonBase__FontSize_Medium {
-	font-size: 14px;
+	font-size: 1.4rem;
```

Comment:
Could these be variables set in a `theme` type configuration file? I'm just assuming that these sizes would be similar to the size of other text in the application.

Reply:
I'll look into that yeah! I was basing it off the text size for Title, though it wasn't exactly 1:1 as far as category :/

Reply:
This is to your discrepancy, if something makes sense feel free to change, but I'm gonna approve nonetheless.